src/lib/workflow_utilities.ml

```diff
@@ -34,11 +34,11 @@ module Remove = struct
   (* This one is dirtier, it does not check its result and uses the `Host.t`
      directly, it should be used only when the `Machine.t` is not available
      (i.e. while defining a `Machine.t`). *)
-  let path_on_host ~host path =
+  let path_on_host ?host path =
```

Comment:
Why this change? 
In Biokepi, the `~host` argument should be always there
(cf. https://github.com/hammerlab/biokepi/pull/170)